network/p2p/libp2pConnector.go

```diff
@@ -18,30 +19,66 @@ import (
 type libp2pConnector struct {
 	backoffConnector *discovery.BackoffConnector
 	host             host.Host
+	log              zerolog.Logger
 }
 
 var _ Connector = &libp2pConnector{}
 
-func newLibp2pConnector(host host.Host) (*libp2pConnector, error) {
+type UnconvertableIdentitiesError struct {
```

Comment:
special error to indicate failure to convert a `flow.Identity` to a libp2p `peer.AddrInfo`